asset/sass/blocks/_component-selector.scss

```diff
@@ -0,0 +1,19 @@
+.component-selector {
+    padding: $padding-large-vertical * 1.5 $padding-large-vertical;
+
+    &__title {
+        color: $text-color-reduced;
+        margin: 0;
+    }
+
+    &__subtitle {
+        color: $text-color-reduced;
+        font-weight: 300;
+        margin-top: 0;
+        margin-bottom: $padding-large-vertical * 1.5;
+    }
+
+    &__control {
+
+    }
```

Comment:
Do we need empty declaration?

Reply:
This empty declaration is not included in the final CSS file and is needed for future adjustments. (Like the [feature request](#issuecomment-389575846) from @weierophinney)